### DISCUSSION TITLE
累計star超過25.4K的OCR神器-PaddleOCR

### DISCUSSION THREAD

**Discussion by xkllkx** [1101-GS3073-C/Python-Community-GS3073-C#480] [Ideas]
同時支援許多AI模型與多種語言的OCR，功能已覆蓋日常生活大部分需求，有興趣的人可以抵選下方連結查看~
[網站測試版](https://www.paddlepaddle.org.cn/hub/scene/ocr)
[PaddleOCR-快速安裝](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_ch/quickstart.md)
[PaddleOCR-github專頁](https://github.com/PaddlePaddle/PaddleOCR)

**Comment by Chenszuhong** [comment]
這感覺是一個很有趣的議題，值得學習

**Comment by brian431** [comment]
文字辨識的運用十分廣泛，在許多人工智慧的應用中都會使用到文字辨識。有這麼一個方便且支持多種語言的工具實在是開發者的一大福音。

**Comment by Ben6uo** [comment]
有這麼一個方便且支持多種語言的工具真的對全球人們的一種福音 可以令所有有興趣人都方便

**Comment by williamyen0623** [comment]
文字辨識很適合用在學習上面，對學生來說真是一大福祉，感謝分享！

**Comment by NcuJerryKao** [comment]
什麼 我竟然沒聽過 感謝
你提供的資料 對初學者而言是福音~

**Comment by Ben6uo** [comment]
太酷了 這對於我來說是新東西 感謝分享

**Comment by 32844583** [comment]
根據我的瞭解，它的識別率通常被認為是高的，並且能夠處理各種不同的文本，包括手寫文本、印刷文本和表格，還支持多種語言。
